website/sidebars.js

```diff
@@ -191,6 +191,7 @@ const sidebarSettings = {
               label: "Install dbt Core",
               link: { type: "doc", id: "docs/core/installation-overview" },
               items: [
+                "docs/core/create-a-python-virtual-environment",
```

Comment:
i know we talked about alphabetizing things but do we think having a 'create a virtual python env' should be the first on the sidebar? before the 'about dbt core'?
![Screenshot 2024-12-19 at 09 25 20](https://github.com/user-attachments/assets/304c1500-e079-441c-a043-41d253b71f65)